packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -153,20 +153,32 @@ Help.propTypes = {
 	}),
 };
 
-function User({ name, renderers, ...props }) {
+function User({ name, firstName, lastName, renderers, ...props }) {
 	const className = classNames(
 		theme['tc-header-bar-action'],
+		theme['tc-header-bar-user'],
 		theme.separated,
 	);
 
+	let displayName = name;
+	if (firstName && lastName) {
+		displayName = (
+			<span className={classNames(theme['user-name'], 'user-name')}>
+				<span className={classNames(theme['user-firstname'], 'user-firstname')}>{firstName}</span>
+				<span className={classNames(theme['user-lastname'], 'user-firstname')}>{lastName}</span>
+			</span>
+		);
+	}
+
 	return (
 		<li className={className}>
 			<renderers.ActionDropdown
 				bsStyle="link"
 				icon="talend-user-circle"
 				pullRight
 				tooltipPlacement="bottom"
-				label={name}
+				tooltipLabel={name}
```

Comment:
THe only difference between `name` and  `displayName` is that `displayName` has `span` to apply the uppercase on the last name. For the `tooltipLabel`, we want the raw text.